bitnami/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -117,9 +144,9 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: ASSET_SYNCER_DB_URL
-              value: {{ template "kubeapps.postgresql.fullname" . }}-headless:{{ default "5432" .Values.postgresql.service.port }}
+              value: {{ printf "%s-headless:%d" (include "kubeapps.postgresql.fullname" .) (int (default "5432" .Values.postgresql.service.ports.postgresql)) | quote }}
             - name: ASSET_SYNCER_DB_NAME
-              value: {{ .Values.postgresql.postgresqlDatabase }}
+              value: {{ .Values.postgresql.postgresqlDatabase | quote }}
```

Comment:
I'm not sure the `quote` pipe is required here, no? Up to you, though. No strong opinion.

Diff of the rendered values in the upstream version and this PR's: 

![image](https://user-images.githubusercontent.com/11535726/159249880-95baeed5-f1b7-4a15-ad0b-8ec78e2b9461.png)

Reply:
Not mandatory to use `quote`, but when handling user input it is good to have since Kubernetes will fail if, for example, Postgres database is numeric.

Not common, but I started applying these protection measures for user inputs after seeing several issues caused by numeric-only namespaces for example. The DB_URL shouldn't need the `quote`, so I'm removing it.